packages/perseus/src/widgets/label-image/__tests__/label-image.test.ts

```diff
@@ -709,4 +714,110 @@ describe("LabelImage", function () {
             });
         });
     });
+
+    describe("getUserInput", () => {
+        it("doesn't include answer in getUserInput from getUserInputMap", () => {
```

Comment:
I think the focus _was_ supposed to be making sure answer was not included, but it's also correct to confirm it's just returning the correct userInput, especially since I didn't write any expect statements specifically checking that answers are absent. I do think it's important to emphasize that the goal is to have answers out of user input, but I guess that will be obvious from the shape of UserInput.